zer0-os/zOS
# Saga Error, Channels app continues to poll when a user is Disconnected which breaks the app

### What does this do?

Fixes the Channel and Conversation Polling issue where if I 'Connect' / Authorize, then 'Disconnect' (I am now Anonymous) the poll continues to attempt to load my Channels and Conversations. The request for Conversations fail since we no longer have permissions to access it.

### How do I test this?

- 'Connect' a User, open the development console, wait two minutes, verify that calls to chatChannels and mine are being made
- 'Disconnect' the User, open the development console, wait two minutes, verify that no calls to chatChannels or mine are being made
